packages/react/src/components/Tree/Tree.tsx

```diff
@@ -90,6 +91,22 @@ class Tree extends AutoControlledComponent<WithAsProp<TreeProps>, TreeState> {
 
   static autoControlledProps = ['activeIndex']
 
+  protected actionHandlers = {
+    expandSiblings: e => {
+      const { items, exclusive } = this.props
+      e.preventDefault()
+      e.stopPropagation()
+
+      if (exclusive) {
```

Comment:
move this check above `e.stopPropagation` & `e.preventDefault`. If we don't handle this keyboard event, we should let it propagate

Reply:
I don't think we should. The event will come from this Tree's title. If we leave it propagate to the parent tree, that one may expand all its siblings, which is not something we want.

This check with `exclusive` is stupid in my opinion, but since we still have this `exclusive` prop for the Tree I had to put it. Maybe we should push to remove it.